shards/modules/core/core.hpp

```diff
@@ -2199,6 +2212,18 @@ struct SeqUser : VariableBase {
     _cell = _target->payload.tableValue.api->tableAt(_target->payload.tableValue, kv);
   }
 
+  SHTypeInfo compose(const SHInstanceData &data) {
+    for (auto &shared : data.shared) {
+      std::string_view vName(shared.name);
+      if (vName == _name && shared.exposed) {
+        SHLOG_ERROR("Exposed variable {} can only be updated using Update.", _name);
+        throw ComposeError("Exposed variables can only be updated using Update.");
+      }
+    }
+
+    return data.inputType;
```

Comment:
Wrong return type, since not all shards that derive from SeqUser return the input type (like count)